src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentPartName.Edit.cshtml

```diff
@@ -0,0 +1,7 @@
+ <h5>
```

Comment:
I don't think this is necessary. It can be kept in the caller template.

Reply:
Moved `H5` and the `DisplayName`/`Desc` of the part to it's own shape  - As most issue related to this is either to remove it from rendering or to keep it's name with it's part when part is placed on Tab

Reply:
Reverted and removed the shape